script/GameObjects/Elements/Core/Light.lua

```diff
@@ -10,11 +10,12 @@ function Entity:addLight(r, g, b)
 end
 
 function Entity:deleteLight(lightRef)
-    assert(GameState.world.currentSystem.lightList)
+    local llist = GameState.world.currentSystem.lightList
```

Comment:
make it lightList

Reply:
The copy seemed to work correctly, but there you go.

Reply:
thanks, its just for readability the double l is not good riddance especiall it would be lList if done correctly